app/views/murals/show.html.erb

```diff
@@ -5,13 +5,21 @@
   <%= image_tag @mural.photo_url %>
   <%= @mural.description %>
   Average Rating: <%= @averageRating%>
+
 </p>
 
 <h2> Reviews </h2>
 <% @reviews.each do |review| %>
   User: <%= review.user.username%>
   Rating: <%= review.rating %>
   Review: <%= review.review%>
+  <% if current_user && current_user.admin? %>
```

Comment:
This is cool, but you could also check if the current user is the OWNER of the review/mural, so they can delete their own stuff